apps/projects/app/components/Panel/ReviewWork/ReviewWork.js

```diff
@@ -24,279 +24,225 @@ import { ipfsAdd } from '../../../utils/ipfs-helpers'
-                    {work.review.user.login}
-                  </SafeLink> {
-                    work.review.accepted ?
-                      'accepted ' + submitterName + '\'s work'
-                      :
-                      'rejected ' + submitterName + '\'s work'
-                  }
-                </Text.Block>
-                {work.review.feedback.length === 0 ?
-                  null
-                  :
-                  <Text.Block style={{ marginBottom: '8px' }}>
-                    {work.review.feedback}
-                  </Text.Block>
-                }
-                <Badge
-                  foreground={theme.textSecondary}
-                  background={theme.contentBorder}
-                >Quality: {work.review.rating}</Badge>
+            ) : (
+              <div css="display: flex; align-items: center">
+                <IconCross color={`${theme.negative}`} css="margin-top: -4px; margin-right: 8px" />
+                <Text color={`${theme.negative}`}>Rejected</Text>
               </div>
-            </ReviewCard>
-          </React.Fragment>
-        ) : (
-          <React.Fragment>
-
-            <FormField
-              label="Quality Rating"
-              required
-              input={
-                <DropDown
-                  items={ratings}
-                  onChange={this.onRatingChange}
-                  active={this.state.rating}
-                />
-              }
-            />
-
-            <FormField
-              label="Feedback"
-              input={
-                <DescriptionInput
-                  name="feedback"
-                  rows="5"
-                  style={{ resize: 'none', height: 'auto' }}
-                  onChange={this.changeField}
-                  value={this.state.feedback}
-                  placeholder="Do you have any feedback to provide the contributor?"
-                  wide
-                />
-              }
-            />
-
-            <ReviewRow>
-              <ReviewButton
-                disabled={this.canSubmit()}
-                mode="negative"
-                onClick={this.onReject}
-              >
-            Reject
-              </ReviewButton>
-              <ReviewButton
-                disabled={this.canSubmit()}
-                mode="positive"
-                onClick={this.onAccept}
-              >
-            Accept
-              </ReviewButton>
-            </ReviewRow>
-          </React.Fragment>
-        )}
-
-      </div>
-    )
-  }
-}
+            )}
+          </div>
+
+          <FieldTitle>Feedback</FieldTitle>
+          <Text.Block style={{ margin: '10px 0' }}>
+            {work.review.feedback.length ? work.review.feedback : 'No feedback was provided'}
+          </Text.Block>
+
+          <Tag
+            uppercase={false}
+            foreground={`${theme.surfaceContentSecondary}`}
+            background={`${theme.border}`}
+          >
+            {'Quality:' + ' ' + ratings[work.review.rating]}
```

Comment:
```jsx
Quality: {ratings[work.review.rating]}
```

Reply:
As above, unfortunately it needs to be as it is now.